packages/web/src/scss/components/Toast/index.html

```diff
@@ -178,85 +315,97 @@ <h2 class="docs-Heading">Content Variations</h2>
+          </div>
         </div>
       </div>
     </div>
 
     <div class="ToastBar ToastBar--inverted">
-      <div class="ToastBar__content">
-        <svg width="20" height="20" aria-hidden="true">
-          <use xlink:href="/assets/icons/svg/sprite.svg#info" />
-        </svg>
-        <div class="ToastBar__message">
-          Message with icon and action
-          <a href="#" class="link-inverted link-underlined">Action</a>
+      <div class="ToastBar__box">
+        <div class="ToastBar__content">
+          <svg width="20" height="20" aria-hidden="true">
+            <use xlink:href="/assets/icons/svg/sprite.svg#info" />
+          </svg>
+          <div class="ToastBar__message">
+            Message with icon and action
+            <a href="#" class="link-inverted link-underlined">Action</a>
+          </div>
         </div>
       </div>
     </div>
 
     <div id="my-toast-variation-dismissible" class="ToastBar ToastBar--inverted ToastBar--dismissible">
-      <div class="ToastBar__content">
-        <div class="ToastBar__message">
-          Dismissible message
+      <div class="ToastBar__box">
+        <div class="ToastBar__content">
+          <div class="ToastBar__message">
+            Dismissible message
+          </div>
         </div>
+        <button
+          type="button"
+          class="Button Button--small Button--square Button--inverted"
+          data-spirit-dismiss="toast"
+          aria-controls="my-toast-variation-dismissible"
+          aria-expanded="true"
+        >
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/assets/icons/svg/sprite.svg#close" />
+          </svg>
+          <span class="accessibility-hidden">Close</span>
+        </button>
       </div>
-      <button
-        type="button"
-        class="Button Button--small Button--square Button--inverted"
-        data-spirit-dismiss="toast"
-        aria-controls="my-toast-variation-dismissible"
-        aria-expanded="true"
-      >
-        <svg width="24" height="24" aria-hidden="true">
-          <use xlink:href="/assets/icons/svg/sprite.svg#close" />
-        </svg>
-        <span class="accessibility-hidden">Close</span>
-      </button>
     </div>
 
     <div id="my-toast-variation-dismissible-with-icon-and-action" class="ToastBar ToastBar--inverted ToastBar--dismissible">
-      <div class="ToastBar__content">
-        <svg width="20" height="20" aria-hidden="true">
-          <use xlink:href="/static/assets/icons/svg/sprite.svg#download" />
-        </svg>
-        <div class="ToastBar__message">
-          Dismissible message with custom icon and action
-          <a href="#" class="link-inverted link-underlined">Action</a>
+      <div class="ToastBar__box">
+        <div class="ToastBar__content">
+          <svg width="20" height="20" aria-hidden="true">
+            <use xlink:href="/static/assets/icons/svg/sprite.svg#download" />
```

Comment:
Is the path correct? Everywhere else it is without `static/`

Reply:
I don't why but both worked… Fixed.